target/linux/ramips/dts/ArcherC5v4.dts

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "tplink,c5-v4", "ralink,mt7620a-soc";
```

Comment:
I'm currently about to change the Archer devices in ramips to have their real names as compatible, in this case tplink,archer-c5-v4. This would be consistent with ath79/ar71xx target.
Consider changing that already.